DataFormats/Luminosity/src/classes_def.xml

```diff
@@ -46,12 +46,12 @@
   </class>
   <class name="std::vector<LumiSummary::L1>"/>
   <class name="std::vector<LumiSummary::HLT>"/>
-  <class name="reco::PixelClusterCounts" ClassVersion="3">
-   <version ClassVersion="3" checksum="1474294271"/>
```

Comment:
Do not remove previous class versions

Reply:
Dear Andrea, I resolved both cases. Commit was pushed